tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Grpc/GrpcDotNet/GrpcAspNetCoreServer/GrpcProtocolHelpersBuildHttpErrorResponseIntegration.cs

```diff
@@ -35,13 +35,19 @@ public class GrpcProtocolHelpersBuildHttpErrorResponseIntegration
         /// <summary>
         /// OnMethodBegin callback
         /// </summary>
+        /// <param name="instance">The instance is always null because is an static class</param>
         /// <param name="response">The HttpResponse value</param>
         /// <param name="httpStatusCode">The HTTP status code value to set</param>
         /// <param name="grpcStatusCode">The GRPC status code</param>
         /// <param name="message">The error message to set</param>
         /// <returns>Calltarget state value</returns>
-        internal static CallTargetState OnMethodBegin(HttpResponse response, int httpStatusCode, int grpcStatusCode, string message)
+        internal static CallTargetState OnMethodBegin<TTarget>(TTarget instance, HttpResponse response, int httpStatusCode, int grpcStatusCode, string message)
```

Comment:
If the instance is always null, should it look like this? 🤔 I get very confused with nullables and generics 😅 

```suggestion
        internal static CallTargetState OnMethodBegin<TTarget>(TTarget? instance, HttpResponse response, int httpStatusCode, int grpcStatusCode, string message)
```

Reply:
done in [4b80233](https://github.com/DataDog/dd-trace-dotnet/pull/5958/commits/4b8023366e77a0e451f4b8c65bc155b22b8e7d4f)